app/drizzle/schema.ts

```diff
@@ -2260,3 +2260,39 @@ export const userRequestRelations = relations(userRequest, ({ one }) => ({
     references: [userData.userId],
   }),
 }));
+
+export const backgroundSchema = mysqlTable(
+  "backgroundSchema",
+  {
+    id: varchar("id", { length: 191 })
+      .primaryKey()
+      .notNull()
+      .default(sql`(UUID())`),
+    schema: json("schema")
+      .$type<{
```

Comment:
Reuse the inferred type from the validated file - so that we only need to maintain it one place :)